common/src/truffle/bytecode.rs

```diff
@@ -83,7 +83,7 @@ impl Bytecode {
     pub fn into_bytes(self) -> Result<Bytes, LinkError> {
         match self.undefined_libraries().next() {
             Some(library) => Err(LinkError::UndefinedLibrary(library.to_string())),
-            None => Ok(Bytes(hex::decode(&self.0).expect("valid hex"))),
+            None => Ok(Bytes(hex::decode(&self.0[2..]).expect("valid hex"))),
```

Comment:
How did this work before?

Reply:
It didn't... I didn't have deploy unit tests so I never caught it until I fixed it for the async example. Here is the true culprit:
https://github.com/gnosis/ethcontract-rs/blob/9fd4f359c110928774ad968620d0e27641f07651/src/contract/deploy.rs#L302

Reply:
See #2